### DISCUSSION TITLE
INDUSTRY SESSION STATUS SHOWING AS INCOMPLETE

### DISCUSSION THREAD

**Discussion by Shrutiind02** [github/india#204] [Students]
Even after uninterruptedly watching an Industry session video, the status is still showing incomplete. Kindly look into this matter.

**Comment by trishna-123** [comment]
Yeah, facing the same problem but no response from them.

**Reply by Shrutiind02** [reply]
Oh I see. Hopefully they'll look into the matter at the earliest.

**Comment by Sharvari230** [comment]
Same here. I have watched all the videos but still its showing incomplete

**Comment by ankitkushawaha1000** [comment]
same here watched 3 - 4 times but still after showing complete then when i refreshed it shows incomplete

**Comment by aarshita02** [comment]
Facing the same issue!

**Reply by Shrutiind02** [reply]
This issue still hasn't been resolved. Kindly look into this matter at the earliest.

**Comment by Sahana8055** [comment]
Even im facing same issue . i gone through it in a single flow 3 times without  jumping ahead ,still the status is incomplete . Hope they resolve this soon

**Comment by TheSuranaverse** [comment]
**FUTURE READY SEPARATE CHANNEL**

For smooth communication & discussion!
Link: https://discord.gg/ceMXzhfaka

Thank you 🙏
